src/main/java/sirius/web/mails/SMTPConfiguration.java

```diff
@@ -89,7 +91,14 @@ public SMTPConfiguration(String host,
     public static SMTPConfiguration fromConfig() {
         return new SMTPConfiguration(smtpHost,
                                      smtpPort,
-                                     smtpProtocol,
+                                     Sirius.getSettings()
+                                           .get("mail.smtp.protocol")
+                                           .getEnum(SMTPProtocol.class)
+                                           .orElseGet(() -> Value.of(Sirius.getSettings()
```

Comment:
Ich finde das als Inline Parameter richtig schrecklich. Würde vorschlagen das in ne Variable auszulagern